app.py

```diff
@@ -37,6 +38,60 @@ def load_data():
     return {}
 
 
+
+
+def create_json_ld(dataset):
+    data = load_data()
+    if dataset not in data:
+        return {}
+    json_ld_dict = {
+      "@context": "https://schema.org/",
+      "@type": "Dataset",
+      "name": data[dataset]['title'],
+      "description": re.sub("<.*>", "",data[dataset]['description']),
+      "url": f"https://data.aifarms.org/view/{ dataset }",
+      #"sameAs": "",
+      #"version": "",
+      "isAccessibleForFree": True,
+      "keywords": [],
```

Comment:
shouldn't this be
```suggestion
      "keywords": data[dataset]['keywords'],
```
one safe way to do this is:
```
      "keywords": data[dataset].get('keywords', []),
```